src/emulator/controller.ts

```diff
@@ -502,14 +501,7 @@ export async function startAll(
     Array.isArray(hostingConfig) ? hostingConfig.some((it) => it.source) : hostingConfig?.source
   ) {
     experiments.assertEnabled("webframeworks", "emulate a web framework");
-    const emulators: EmulatorInfo[] = [];
-    if (experiments.isEnabled("webframeworks")) {
-      for (const e of EMULATORS_SUPPORTED_BY_UI) {
-        const info = EmulatorRegistry.getInfo(e);
-        if (info) emulators.push(info);
-      }
-    }
-    await prepareFrameworks(targets, options, options, emulators);
+    await prepareFrameworks(targets, options, options);
```

Comment:
We will probably need to explicitly test that the frameworks stuff works. It looks fine, but it's sensitive to disruption right now.

Reply:
As discussed offline, I've reverted my changes to frameworks and added a comment on the code in question.